lib/agenda/enable.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const debug = require('debug')('agenda:enable');
+
+/**
+ * Enables any jobs matching the passed MongoDB query, and removes them from the database.
```

Comment:
"and removes them from the database" ?
I don't see that happening in the code. :) Perhaps, the comment needs an update?